index.html

```diff
@@ -43,6 +43,231 @@ <h1 class="primary-text">
+                <div class="pr-call-to-action-box">
+                    <a href="#" class="pr-call-to-action-button"> See Project</a>
+                </div>
+            </div>
+        </div>
+        <!-- second card-->
+        <div class="card-two">
+            <!--portfolio image two -->
+            <div class="pr-photo2">
+                <img class="portfolio-img" src="./images/photos/portfolio-pic-2.png" alt="2nd portfolio photo">
+            </div>
+            <!--portfolio details two -->
+           <div class="pr-details2"> 
+                <!--project title-->
+                <h2 class="pr-title">
+                    Multi-Post Stories
+                </h2>
+                <!--project type-->
+                <ul class="pr-type">
+                    <li class="no-style"> CANOPY </li>
+                    <li> Backend Dev </li>
+                    <li> 2015 </li>
+                </ul>
+                <!--project description-->
+                <p class="pr-description ">
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.
+                </p>
+                <!--tech tags-->
+                <ul class="pr-tech-tags no-style">
+                    <li><a href="#" class="pr-tag"> html </a></li>
+                    <li><a href="#" class="pr-tag"> css </a></li>
+                    <li><a href="#" class="pr-tag"> javascript </a></li>
+                </ul>
+                <!--action button -->
+                <div class="pr-call-to-action-box">
+                    <a href="#" class="pr-call-to-action-button"> See Project</a>
+                </div>
+            </div>
+        </div>
+        <!-- third card -->
+        <div class="card-three">
+            <!--portfolio image three -->
+            <div class="pr-photo3">
+                <img class="portfolio-img" src="./images/photos/portfolio-pic-3.png" alt="3rd portfolio photo">
+            </div>
+            <!--portfolio details three -->
+           <div class="pr-details3"> 
+                <!--project title-->
+                <h2 class="pr-title">
+                    Tonic
+                </h2>
+                <!--project type-->
+                <ul class="pr-type">
+                    <li class="no-style"> CANOPY </li>
+                    <li> Backend Dev </li>
+                    <li> 2015 </li>
+                </ul>
+                <!--project description-->
+                <p class="pr-description ">
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.
+                </p>
+                <!--tech tags-->
+                <ul class="pr-tech-tags no-style">
+                    <li><a href="#" class="pr-tag"> html </a></li>
+                    <li><a href="#" class="pr-tag"> css </a></li>
+                    <li><a href="#" class="pr-tag"> javascript </a></li>
```

Comment:
It would be better if you remove the `a` from the `li` since it's not something you're going to add a link to.